test/functional/checkupdate/chk-update-slow-server.bats

```diff
@@ -28,15 +29,11 @@ global_teardown() {
 
 @test "check-update with a slow server" {
 
-	# Pre-req: create a web server that can serve as a slow content download server
-	slow_opts="-p $TEST_NAME/target-dir -F staging -u http://localhost:$PORT/"
-
-	# test
-	run sudo sh -c "$SWUPD check-update $slow_opts"
+	run sudo sh -c "$SWUPD check-update $SWUPD_OPTS_HTTP_NO_CERT"
 	assert_status_is 0
 	expected_output=$(cat <<-EOM
 		Current OS version: 10
-		There is a new OS version available: 99990
+		There is a new OS version available: 20
```

Comment:
The reason the original slow server used version `99990` was to have the test web server write one byte at a time, with a small delay between writes to make it seem that the server was responding, but slowly.

I'm not sure the revised slow server does this anymore (see other comment below).